projects/vdk-core/plugins/vdk-greenplum/README.md

```diff
@@ -0,0 +1,33 @@
+This plugin allows vdk-core to interface with and execute queries against a Greenplum database.
+
+# Usage
+
+Run
+```bash
+pip install vdk-greenplum
+```
+
+After this, data jobs will have access to a Greenplum database connection, managed by Versatile Data Kit SDK.
+
+If it is the only database plugin installed , vdk would automatically use it.
+Otherwise, users need to set VDK_DB_DEFAULT_TYPE=GREENPLUM as an environment variable or set 'db_default_type' option in the data job config file (config.ini).
+
+For example
+
+```python
+    def run(job_input: IJobInput):
+        job_input.execute_query("select 'Hi Greenplum!'")
+```
+
+# Configuration
+
+Run vdk config-help - search for those prefixed with "GREENPLUM_" to see what configuration options are available.
```

Comment:
This makes me think whether we should extend config-help to allow config variable sorting. If one has 15 or 20 plugins installed, it might be very annoying to have to scroll through them to find the one you're looking for.

Reply:
This is obviously unrelated to this PR.